Tasks/Gomolinskaya/Task 4/Task 4.py

```diff
@@ -0,0 +1,24 @@
+s = int (input ("Введите число больше 35 \n"))
+while True:
+    if s <= 35:
+        print (' Неверное число')
+    else:
+        break
+print ('Принято')    
+
+with open ("D:/Гомолинская/Md-PT1-60-23/Tasks/Gomolinskaya/Task 4/text.txt", "r", encoding = 'utf-8') as f:
+    for line in f:  
+        count = 0
+        row_size = len(line)
+        with open ("D:/Гомолинская/Md-PT1-60-23/Tasks/Gomolinskaya/Task 4/n_text.txt", "a" , encoding = 'utf-8') as n:
```

Comment:
please refrain from using full (non-relative) paths in any of your program which will be posted somewhere, it may lead to errors for those users who do not have similar folder structure